src/specs/css-loader.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { lorem } from './util';
+
+
+const IS_BROWSER = typeof(window) !== 'undefined';
+if (IS_BROWSER) {
+  require('./css-loader.css');
+}
```

Comment:
@philcockfield if we no longer load the js files on the server, do we need these hacks?